Firebase/Database/Persistence/FLevelDBStorageEngine.m

```diff
@@ -216,14 +220,25 @@ + (NSString *) firebaseDir {
     #endif
 }
 
-- (APLevelDB *)createDB:(NSString *)name {
+- (APLevelDB *)createDB:(NSString *)dbName {
     NSError *err = nil;
-    NSString *path = [self.basePath stringByAppendingPathComponent:name];
+    NSString *path = [self.basePath stringByAppendingPathComponent:dbName];
     APLevelDB *db = [APLevelDB levelDBWithPath:path error:&err];
-    if(err) {
-        NSString *reason = [NSString stringWithFormat:@"Error initializing persistence: %@", [err description]];
-        @throw [NSException exceptionWithName:@"FirebaseDatabasePersistenceFailure" reason:reason userInfo:nil];
+
+    if (err) {
+        FFWarn(@"I-RDB076017", @"Failed to read database persistence file '%@': %@",
+               dbName, [err localizedDescription]);
```

Comment:
There is already an existing error that the actual deleting code logs. I upgraded that one to a warning and made this an error.